src/server.h

```diff
@@ -1312,6 +1312,8 @@ struct redisServer {
     long long stat_expired_time_cap_reached_count; /* Early expire cycle stops.*/
     long long stat_expire_cycle_time_used; /* Cumulative microseconds used. */
     long long stat_evictedkeys;     /* Number of evicted keys (maxmemory) */
+    long long stat_eviction_exceeded_time;  /* Total time over the memory limit */
+    long long stat_last_eviction_exceeded_time; /* Timestamp of current eviction start */
```

Comment:
i think `total` should be added to the variable, to make it clearer.
```suggestion
    long long stat_total_eviction_exceeded_time;  /* Total time over the memory limit */
    long long stat_last_eviction_exceeded_time; /* Timestamp of current eviction start */
```